### DISCUSSION TITLE
Ubuntu Impish Indri (Ubuntu Studio distro) question

### DISCUSSION THREAD

**Discussion by BriocheBaps** [enzo1982/freac#284] [General Discussion]
Please, forgive my ignorance I am new to Linux and have installed Freac to manage my ripped music and to rip music in the future. I am having a problem adding a files to process and transcode. I have my OS installed on my NVME drive, but I am using a slower secondary SSD drive for storage of music, samples, files etc. I can't seem to get Freac to access the SSD, I have tried a usermod command - I typed "sudo usermod -aG freac $USER" of course replacing $USER with my username, I was returned the error that the group "freac" does not exist. Do I need to mess with the Chown command and "own" the SSD? Am I missing something very simple re: allowing Freac to access the SSD? 

I am very new to Linux, command line, etc. and would be grateful for a friendly pointer. I'm 38 and used Windows forever, but W11 and the general data scraping nature of Windows has led me to move to Linux. I am reading a lot and trying my hardest to aquaint myself to the paradigm of Ubuntu. Sorry if this is like some elementary level nonsense but with limited knowledge I don't know what terms to use when searching the internet for a guide. Thanks.

**Comment by enzo1982** [comment]
Did you install fre:ac from the Ubuntu Software store? That version is running in a sandbox that restricts which places fre:ac can access.

You can use the [fre:ac AppImage](https://github.com/enzo1982/freac/releases/) instead which should be able to access all storage on your computer.

**Reply by BriocheBaps** [reply]
I am indeed running a copy grabbed from the "Discover" store. I'll grab the AppImage later and give it a go.

Edit: thank you!
